processing.go

```diff
@@ -78,6 +90,13 @@ func grabTarget(input ScanTarget, m *Monitor) []byte {
 	moduleResult := make(map[string]ScanResponse)
 
 	for _, scannerName := range orderedScanners {
+		defer func(name string) {
+			if e := recover(); e != nil {
+				log.Errorf("Panic on scanner %s when scanning target %s", scannerName, input.String())
+				// Bubble out original error (with original stack) in lieu of explicitly logging the stack / error
+				panic(e)
```

Comment:
So we actually get the original stack trace here, we just get to shove in a log statement? That's awesome!

Reply:
I was suspicious too, so I tried it out, and it works as advertised.